site/public/js/ta-grading.js

```diff
@@ -406,7 +406,7 @@ function adjustGradingPanelHeader () {
   }
   // From the complete content remove the height occupied by navigation-bar and panel-header element
   // 6 is used for adding some space in the bottom
```

Comment:
Would this comment not need to be modified or removed?

Reply:
You are correct.  I had the change on my branch and then it got deleted amongst some other Git issues.